lib/sidekiq/dynamic_queues/attributes.rb

```diff
@@ -109,7 +122,24 @@ def expand_queues(queues)
           end
         end
 
-        return matched_queues.collect { |q| "queue:#{q}" }.uniq
+        return matched_queues.sort
+      end
+
+      def expand_queues_for_display(queues)
```

Comment:
I'm not sure I understand what you mean but wouldn't calling `.uniq` essentially do the same thing (minus the setting the -1 key) while reducing the complexity of this function/making it easier to read?